org/ystia/yorc/yorc/pub/types.yaml

```diff
@@ -0,0 +1,203 @@
+tosca_definitions_version: alien_dsl_2_0_0
+#
+# Copyright 2018 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+metadata:
+  template_name: org.ystia.yorc.pub
+  template_version: 2.1.0-SNAPSHOT
+  template_author: yorc
+
+imports:
+  - tosca-normative-types:1.0.0-ALIEN20
+  - org.ystia.yorc.experimental.consul.pub:2.1.0-SNAPSHOT
+  - org.ystia.ansible.pub:2.1.0-SNAPSHOT
+  - org.ystia.terraform.pub:2.1.0-SNAPSHOT
+
+node_types:
+  org.ystia.yorc.pub.infrastructure.OpenStackConfig:
+    derived_from: tosca.nodes.Root
+    description: |
+      Openstack Infrastructure configuration for Yorc
+    abstract: true
+    properties:
+      auth_url:
+        description: "The authentication url for OpenStack (should be the Keystone endpoint ie: http://your-openstack:5000/v2.0)."
+        type: string
+        required: true
+      tenant_name:
+        description: "The OpenStack tenant name to use."
+        type: string
+        required: true
+      user_name:
+        description: "The OpenStack user name to use."
+        type: string
+        required: true
+      password:
+        description: "The OpenStack password to use."
+        type: string
+        required: true
+      private_network_name:
+        description: "The name of private network to use as primary adminstration network between Yorc and Compute instances. It should be a private network accessible by this instance of Yorc."
+        type: string
+      default_security_groups:
```

Comment:
Caution: for properties definitions the default value for the `required` attribute is  `true`.
I think we should make `default_security_groups` optional.